packages/launchpad-navigator/src/floating-navigator/index.tsx

```diff
@@ -1,6 +1,7 @@
-import { Card, Button, Gridicon } from '@automattic/components';
+import { Card, Gridicon } from '@automattic/components';
 import { LaunchpadNavigator } from '@automattic/data-stores';
 import { DefaultWiredLaunchpad } from '@automattic/launchpad';
+import { Button } from '@wordpress/components';
```

Comment:
It looks like we're missing `@wordpress/components` as a dependency at the package level -- we should add the dependency explicitly to make sure it's installed/available.